src/current/v23.1/debezium.md

```diff
@@ -0,0 +1,94 @@
+---
+title: Migrate Data with Debezium
+summary: Use Debezium to migrate data to a CockroachDB cluster.
+toc: true
+docs_area: migrate
+---
+
+[Debezium](https://debezium.io/) is a self-hosted distributed platform that can read data from a variety of sources and import it into Kafka. You can use Debezium to [migrate data to CockroachDB](#migrate-data-to-cockroachdb) from another database that is accessible over the public internet.
+
+As of this writing, Debezium supports the following database [sources](https://debezium.io/documentation/reference/stable/connectors/index.html):
+
+- MongoDB
```

Comment:
I'm not sure we should list the specific ones unless we are committing to keep the list up to date. Maybe we should mention that it supports many popular database such as ... maybe the first 5 here?

Reply:
It seems to be the precedent for all the other migration pages, was mostly going off these:
https://www.cockroachlabs.com/docs/v23.1/striim
https://www.cockroachlabs.com/docs/v23.1/aws-dms
https://www.cockroachlabs.com/docs/v23.1/qlik

Reply:
Ahh ok, makes sense. I didn't realize there was a precedent.